pages/common/kubetail.md

```diff
@@ -0,0 +1,19 @@
+# kubetail
+
+> Utility to tail multiple Kubernetes pod logs at the same time.
+
+- To tail the logs of multiple pods (which name starts with "my_app") in one go simply do:
```

Comment:
Perhaps change each to start with `Tail` rather than `To tail`?
```suggestion
- Tail the logs of multiple pods (which name starts with "my_app") in one go:
```